src/App.js

```diff
@@ -1,8 +1,76 @@
-function App() {
+import { useState } from "react";
+import sunny from "./sunny.png"
+import rainy from "./rainy.png"
+import semi from "./semi.png"
+import cloudy from "./cloudy.png"
+
+var picture= sunny;
+
+
+function App() { 
+
+
+var APIdata 
+const [data,setData ] = useState({} );
+const [location,setLocation ] = useState("" );
+
+const apiKey = "ffcc3b1d6839acddfd18b10b824cdae1";
+const apiUrl=  "https://api.openweathermap.org/data/2.5/weather?&units=metric&q=";
+
+ const checkWeather = async () => {
+ const response = await fetch(apiUrl + location +'&appid=' + apiKey);
+  APIdata = await response.json();
+ setData(APIdata)
+ console.log(APIdata);
+  
+}
+
+ 
+const getData = (event) => {
+  if(event.key === 'Enter'){
+
+    checkWeather();
+    if (data && data.clouds && typeof data.clouds.all !== 'undefined') {
+      if (data.clouds.all > 39 && data.clouds.all < 70) {
+        picture = semi;
```

Comment:
instead of 'var picture' you can use 
const [picture, setPicture] = useState()
and on line 35 'setPicture(semi)'